docs/docs/api-reference.md

```diff
@@ -118,7 +125,7 @@ Default `true`. Setting `reactive` to false will improve game logic performance,
 Whenever a player tries to do an action that is not allowed, the action handler should reject it by calling `throw Rune.invalidAction()` which will cancel the action and roll back any local optimistic updates.
 
 This is completely safe to do and can be used throughout your game. For instance, it is used in the [Tic Tac Toe example](https://github.com/rune/rune/blob/staging/examples/tic-tac-toe/logic.js) to ensure that players only can make a move when it is their turn.
-
+### 
```

Comment:
Just checking, is this empty heading intentional?

Reply:
Good catch thank you